openslides/utils/autoupdate.py

```diff
@@ -71,41 +69,47 @@ def on_close(self):
 
     def handle_rest_request(self, response):
         """
-        Handler that is called when the rest api responds.
+        Sends data to the client of the connection instance.
 
-        Sends the response.body to the client.
+        This method is called after succesful response of AsyncHTTPClient().
+        See send_object().
         """
-        # TODO: update cookies
+        # TODO: Update cookies of the client.
+        url_parts = response.request.url.split('/')
+        data = {
+            'url': response.request.url,
+            'name': '%s/%s' % (url_parts[-4], url_parts[-3]),
+            'data': json.loads(response.body.decode()),}
+        # TODO: Check and handle other status codes.
         if response.code == 200:
-            self.send(response.body)
-
-    @classmethod
-    def send_updates(cls, data):
-        # TODO: use a bluk send
-        for waiter in cls.waiters:
-            waiter.send(data)
+            self.send(data)
 
     @classmethod
     def send_object(cls, object_url):
         """
-        Send OpenSlides objects to all connected clients.
+        Sends an OpenSlides object to all connected clients.
 
-        First, receive the object from the OpenSlides ReST API.
+        First, receive the object from the OpenSlides REST api using the given
+        object_url.
         """
         for waiter in cls.waiters:
-            # Get the object from the ReST API
             http_client = AsyncHTTPClient()
-            headers = HTTPHeaders()
+
             # TODO: read to python Morselcookies and why "set-Cookie" does not work
+            headers = HTTPHeaders()
             request_cookies = waiter.request_info.cookies.values()
             cookie_value = ';'.join("%s=%s" % (cookie.key, cookie.value)
                                     for cookie in request_cookies)
             headers.parse_line("Cookie: %s" % cookie_value)
 
+            # TODO: Use host and port as given in the start script
+            host_and_port = settings.OPENSLIDES_HOST_AND_PORT or 'localhost:8000'
```

Comment:
tornado is also a part of openslides. If the tornado runs on one server and django on another, this config value should only specify the domain that tornado should use to connect to django. Could you find a better name for it?